src/components/Router/StepComponentMap.js

```diff
@@ -0,0 +1,53 @@
+import { h, Component } from 'preact'
+import Welcome from '../Welcome'
+import Select from '../Select'
+import {FrontDocumentCapture, BackDocumentCapture, FaceCapture} from '../Capture'
+import Complete from '../Complete'
+
+export const steps = (stepOptions, documentType) => {
+  const mapOptions = stepOption => stepOptionToStep(stepOption, documentType)
+  return shallowFlatten(stepOptions.map(mapOptions))
+}
+
+export const components = (stepList, componentOptions) => {
```

Comment:
this name doesn't really tell me much, it's also quite general. something like `stepListToComponentsList` would be more descriptive.